src/Cake.Common/Tools/OpenCover/OpenCoverAliases.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using Cake.Core;
+using Cake.Core.Annotations;
+
+namespace Cake.Common.Tools.OpenCover
+{
+    /// <summary>
+    /// Contains functionality related to OpenCover.
+    /// </summary>
```

Comment:
I agree that there needs to be some kind of convention for this. It would be nice if we supported the (MSDN) [`<list>` tags](https://msdn.microsoft.com/en-us/library/y3ww3c7e.aspx).

We could address this in another PR.